isis/src/qisis/apps/ipce/ipce.cpp

```diff
@@ -35,7 +35,10 @@ using namespace std;
 using namespace Isis;
 
 int main(int argc, char *argv[]) {
-
+  if (getenv("ISISROOT") == NULL || QString(getenv("ISISROOT")) == "") {
+    std::cerr << "Please set ISISROOT before running any Isis applications" << std::endl;
```

Comment:
Can this message be pulled into a different, common class that is shared with all of the qisis apps? That way there isn't duplicated code?